microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -2724,6 +2717,7 @@ See also:
 Analyze and debug performance, handle performance-related events, and manage memory usage to increase the responsiveness of your app.
```

Comment:
I don't think runtime selection should be in performance and debugging. 
My suggestion is to create:
- a new Environmental Options section, and we put both User Data and Runtime selection under it.
- Create a new Runtime Selection section

Reply:
```
<!-- ====================================================================== -->
## Environment options

**Subsections below:**
* [User data](#user-data)
* [Runtime selection](#runtime-selection)

...

<!-- ------------------------------ -->
#### Runtime selection

The Runtime Selection feature supports prerelease testing and self-hosting.
To specify a search order for browser preview channels, use the
`CoreWebView2EnvironmentOptions.ChannelSearchKind` property.  
To specify which browser preview channels are searched for during environment creation, use the
`CoreWebView2EnvironmentOptions.ReleaseChannels` property.

See also:
* [Test upcoming APIs and features](../how-to/set-preview-channel.md)
<!-- todo: link, after PR https://github.com/MicrosoftDocs/edge-developer/pull/3053 is merged:
* [Prerelease testing using preview channels](../how-to/prerelease-testing.md)
* [Self-host by deploying preview channels](../how-to/self-hosting.md)
-->
```